packages/hardware-ledger/src/transformers/votingProcedures.ts

```diff
@@ -0,0 +1,114 @@
+import { Cardano } from '@cardano-sdk/core';
+import {
+  CommitteeKeyHashVoter,
+  CommitteeScriptHashVoter,
+  DRepKeyHashVoter,
+  DRepScriptHashVoter,
+  StakePoolKeyHashVoter,
+  Vote,
+  VoteOption,
+  Voter,
+  VoterType,
+  VoterVotes
+} from '@cardano-foundation/ledgerjs-hw-app-cardano/dist/types/public';
+
+/**
+ * Maps a Voter to a Voter from the LedgerJS public types.
+ *
+ * @param {Cardano.Voter} voter - The voter object defined in Core types.
+ * @returns {Voter} Corresponding Voter object for use with LedgerJS.
+ */
+export const mapVoterToLedgerVoter = (voter: Cardano.Voter): Voter => {
+  switch (voter.__typename) {
+    case 'ccHotKeyHash':
+      return {
+        keyHashHex: voter.credential.hash,
+        type: VoterType.COMMITTEE_KEY_HASH
+      } as CommitteeKeyHashVoter;
+
+    case 'ccHotScriptHash':
+      return {
+        scriptHashHex: voter.credential.hash,
+        type: VoterType.COMMITTEE_SCRIPT_HASH
+      } as CommitteeScriptHashVoter;
```

Comment:
wasn't mapped out in this [pr](https://github.com/input-output-hk/cardano-js-sdk/pull/1211/files#diff-cbe9d96fa74567534f1417be16c9041135a5e2431286755f3323c17f65339366R32) , @mchappell do you recall what was the reason behind that?

Reply:
That was on my side, I didn't think it was possible to use those certs, but it looks like they are valid

Reply:
do I need to change something here? @vetalcore